explainer.md

```diff
@@ -0,0 +1,369 @@
+# Secure Payment Confirmation explained
+
+## tl;dr
+
+**Secure Payment Confirmation** is a proposed Web API that enables the use of
+biometric authentication in payment flows on the web, building on top of
+[WebAuthn]. The goal is to provide strong, low-friction authentication of a
+customer in a payment context, whilst still preserving user privacy better than
+current fingerprinting-based methods.
+
+See also:
+  - [Specification](https://w3c.github.io/secure-payment-confirmation/), the formal draft spec.
+  - [Scope document](https://github.com/w3c/secure-payment-confirmation/blob/main/scope.md)
+  - [Requirements document](https://github.com/w3c/secure-payment-confirmation/blob/main/requirements.md)
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Contents
+
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposed Solution: Secure Payment Confirmation](#proposed-solution-secure-payment-confirmation)
+  - [Proposed APIs](#proposed-apis)
+    - [Creating a credential](#creating-a-credential)
+      - [Creating a SPC credential in a cross-origin iframe](#creating-a-spc-credential-in-a-cross-origin-iframe)
+    - [Authenticating a payment](#authenticating-a-payment)
+- [Other Considerations](#other-considerations)
+  - [Initial Experimentation with Stripe](#initial-experimentation-with-stripe)
+  - [Are SPC credentials identical to WebAuthn credentials?](#are-spc-credentials-identical-to-webauthn-credentials)
+  - [Why use the PaymentRequest API](#why-use-the-paymentrequest-api)
+- [Alternatives Considered](#alternatives-considered)
+  - [Traditional WebAuthn](#traditional-webauthn)
+  - [Delegated Authentication](#delegated-authentication)
+- [Security Considerations](#security-considerations)
+  - [Enrollment in cross-origin iframes](#enrollment-in-cross-origin-iframes)
+  - [Cross-origin authentication ceremony](#cross-origin-authentication-ceremony)
+  - [Merchant-supplied data](#merchant-supplied-data)
+- [Privacy Considerations](#privacy-considerations)
+  - [Probing](#probing)
+  - [Credential-sharing](#credential-sharing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+Online payments are usually a 3 party interaction:
+
+* The Merchant (e.g. an online web store)
+* The Customer (e.g. the user buying from an online web store)
+* The ??? (e.g. the bank that issued the payment instrument being used)
```

Comment:
Not sure on the best term to use here. The Issuer ?

Reply:
Either issuer or RP could work?

Reply:
A variety of relying parties might use SPC (including the merchant in delegated authentication flows).  So what about:

* The user's bank (that issues the instrument used to make the payment).

Reply:
I'm ok with Bank, I just wasn't sure if it's appropriate when other entities may be behind the instrument in theory. But if Bank is generally accepted, then Bank sgtm.

Reply:
We might want to opt for Account Provider, which is a term that comes from Open Banking. 
There are a number of organizations that perform banking, but who prefer not to be called banks. These include Credit Unions and Issuers.

Reply:
+1 to Account Provider. May still be useful to say 'account provider (e.g., issuing bank)'

Reply:
Sounds like consensus to me :D. Done!